release/models/acl/openconfig-packet-match.yang

```diff
@@ -274,6 +281,16 @@ module openconfig-packet-match {
         "Source IPv4 address prefix.";
     }
 
+    leaf source-address-prefix-list {
+      type leafref {
+         path "/oc-sets:defined-sets/oc-sets:ipv4-prefix-sets"
```

Comment:
This path looks good to me -- I think we can review as a group whether this should be something that is in another top-level container. `defined-sets` seems reasonable.